meta-lmp-bsp/recipes-bsp/u-boot/u-boot-fio/apalis-imx6-sec/lmp.cfg

```diff
@@ -0,0 +1,100 @@
+CONFIG_SPL_SEPARATE_BSS=y
+CONFIG_MX6_OCRAM_256KB=y
+CONFIG_SPL_SIZE_LIMIT=200704
+CONFIG_SPL_DM=y
+CONFIG_SPL_DM_GPIO=y
+CONFIG_SPL_FIT_SIGNATURE=y
+CONFIG_SPL_FIT_SIGNATURE_STRICT=y
+CONFIG_SPL_FIT_PRINT=y
+CONFIG_SPL_LOAD_FIT=y
+CONFIG_SPL_SHA256_SUPPORT=y
+CONFIG_SPL_CRYPTO_SUPPORT=y
+CONFIG_SPL_HASH_SUPPORT=y
+CONFIG_SPL_WATCHDOG_SUPPORT=y
+CONFIG_SPL_OPTEE=y
+CONFIG_SPL_OF_CONTROL=y
+CONFIG_CMD_SECONDARY_BOOT=y
+CONFIG_SECONDARY_BOOT_RUNTIME_DETECTION=y
+CONFIG_SECONDARY_BOOT_SECTOR_OFFSET=0x800
+CONFIG_SYS_MMCSD_RAW_MODE_U_BOOT_SECTOR=0x100
+CONFIG_WDT=y
+CONFIG_WATCHDOG=y
+CONFIG_IMX_WATCHDOG=y
+CONFIG_DM_RESET=y
+CONFIG_SYSRESET=y
+CONFIG_SYSRESET_PSCI=y
+# CONFIG_SPL_ENV_SUPPORT is not set
+# CONFIG_SPL_LEGACY_IMAGE_SUPPORT is not set
+# CONFIG_SPL_EFI_PARTITION is not set
+# CONFIG_SPL_PARTITION_UUIDS is not set
+# CONFIG_SPL_BLK is not set
+# CONFIG_SPL_DM_SERIAL is not set
+# CONFIG_SPL_DM_USB is not set
+# CONFIG_SPL_FIOVB is not set
+# CONFIG_SPL_I2C_SUPPORT is not set
+# CONFIG_SPL_USB_HOST_SUPPORT is not set
+# CONFIG_SPL_USB_GADGET is not set
+# CONFIG_SPL_DM_MMC is not set
+CONFIG_IMX_HAB=y
+# CONFIG_CMD_DEKBLOB is not set
+CONFIG_TEE=y
+CONFIG_OPTEE=y
+CONFIG_OF_LIBFDT_OVERLAY=y
+CONFIG_OPTEE_TZDRAM_SIZE=0x02000000
+CONFIG_OPTEE_TZDRAM_BASE=0x4E000000
+CONFIG_OPTEE_SKIP_LOWLEVEL_INIT=y
+CONFIG_SYS_L2CACHE_OFF=y
+CONFIG_FIT=y
+CONFIG_FIT_SIGNATURE=y
+CONFIG_FIT_SIGNATURE_STRICT=y
+CONFIG_FIT_VERBOSE=y
+# CONFIG_LEGACY_IMAGE_FORMAT is not set
+CONFIG_FIOVB=y
+CONFIG_CMD_FIOVB=y
+CONFIG_CMD_XIMG=y
+CONFIG_IMX_WATCHDOG=y
+CONFIG_SUPPORT_EMMC_RPMB=y
+CONFIG_DM_I2C_GPIO=y
+# CONFIG_ENV_IS_IN_MMC is not set
+CONFIG_ENV_IS_IN_FAT=y
+CONFIG_ENV_FAT_INTERFACE="mmc"
+CONFIG_ENV_FAT_DEVICE_AND_PART="0:1"
```

Comment:
^^ These 2 settings are what get removed.

Reply:
@mike-scott 
```
--- a/meta-lmp-bsp/recipes-bsp/u-boot/u-boot-fio/apalis-imx6-sec/lmp.cfg
+++ b/meta-lmp-bsp/recipes-bsp/u-boot/u-boot-fio/apalis-imx6-sec/lmp.cfg
@@ -56,9 +56,8 @@ CONFIG_IMX_WATCHDOG=y
 CONFIG_SUPPORT_EMMC_RPMB=y
 CONFIG_DM_I2C_GPIO=y
 # CONFIG_ENV_IS_IN_MMC is not set
-CONFIG_ENV_IS_IN_FAT=y
-CONFIG_ENV_FAT_INTERFACE="mmc"
-CONFIG_ENV_FAT_DEVICE_AND_PART="0:1"
+# CONFIG_ENV_IS_IN_FAT is not set
+CONFIG_ENV_IS_NOWHERE=y
```
is this ok?